src/core/layout/qgslayout.h

```diff
@@ -303,14 +303,16 @@ class CORE_EXPORT QgsLayout : public QGraphicsScene, public QgsExpressionContext
     /**
      * Returns the topmost layout item at a specified \a position. Ignores paper items.
      * If \a ignoreLocked is set to TRUE any locked items will be ignored.
+     * \since QGIS 3.32 searchRadius parameter was added
      */
-    QgsLayoutItem *layoutItemAt( QPointF position, bool ignoreLocked = false ) const;
+    QgsLayoutItem *layoutItemAt( QPointF position, bool ignoreLocked = false, double searchRadius = 0 ) const;
```

Comment:
This should rather be "searchTolerance", it's not a radius

Reply:
Agreed. Although "radius" is wrongly used in many other places
https://github.com/qgis/QGIS/blob/5e79276e732b0279a75d32be85f94876eb3ea5a3/src/gui/qgsmaptoolidentify.cpp#L599-L600